win32ss/gdi/ntgdi/freetype.c

```diff
@@ -206,6 +231,91 @@ BYTE FASTCALL IntCharSetFromCodePage(UINT uCodePage)
     return DEFAULT_CHARSET;
 }
 
+static BOOL
+SubstituteFontRecurse(LOGFONTW* pLogFont);
+
+static PFONTLINK
+FontLink_Create(PFONTLINK_CHAIN pChain, const LOGFONTW *plfBase, LPWSTR pszLink)
+{
+    LPWSTR pch1, pch2;
+    LOGFONTW lf;
+    PFONTLINK pLink;
+
+    pch1 = wcschr(pszLink, L',');
+    if (!pch1)
+        return NULL;
+
+    ++pch1; /* Next of comma */
```

Comment:
Instead of having these comments ("next comma", "cut off"...), maybe explain instead why you are searching for the commas in the pszLink and doing this stuff.

Reply:
Fixed.